{"name": "test_sign_up_negative_8[4]", "status": "failed", "statusDetails": {"message": "AssertionError: assert \"{'username':...must match']}\" == 'Incorrect us...ds must match'\n  - Incorrect username length and passwords must match\n  + {'username': ['Incorrect username length'], 'password': ['Passwords must match']}", "trace": "self = <test_ui.TestUI object at 0x7f267c5bcdc0>\nget_fake_user_info = ('mihalovdanila', 'HJ2yCg)zh@', 'boris32@mail.ru')\nlen_username = 4\nhome_page_object = <ui.pages.home_page.HomePage object at 0x7f267c652670>\nreg_page_object = <ui.pages.reg_page.RegPage object at 0x7f267c652c10>\n\n    @allure.feature('UI tests')\n    @allure.story('Sign up page story')\n    @pytest.mark.parametrize('len_username', [1, 2, 3, 4, 5])\n    def test_sign_up_negative_8(self, get_fake_user_info, len_username, home_page_object, reg_page_object):\n        \"\"\"Проверка регистрации пользователя с одновременно несовпадающими полями password, confirm_password и невалидным полем username\"\"\"\n        self.logger.start_logging(\n            f'./project/app_logs/{inspect.getframeinfo(inspect.currentframe()).function}_ui_log_{len_username}.txt')\n        with allure.step('Получаем информацию о невалидном пользователе'):\n            username, password, email = get_fake_user_info\n            if len(username) < len_username:\n                username *= len_username\n            username = username[:len_username]\n        with allure.step('Получаем объект страницы регистрации'):\n            reg_page = self.login_page.sign_up(reg_page_object)\n        with allure.step('Регистрируемся и проверяем валидное отбражение ошибки на странице регистрации'):\n            try:\n>               reg_page.sign_up(username, email, password, home_page_object, wrong_password=password + 'qwe')\n\nemail      = 'boris32@mail.ru'\nget_fake_user_info = ('mihalovdanila', 'HJ2yCg)zh@', 'boris32@mail.ru')\nhome_page_object = <ui.pages.home_page.HomePage object at 0x7f267c652670>\nlen_username = 4\npassword   = 'HJ2yCg)zh@'\nreg_page   = <ui.pages.reg_page.RegPage object at 0x7f267c652c10>\nreg_page_object = <ui.pages.reg_page.RegPage object at 0x7f267c652c10>\nself       = <test_ui.TestUI object at 0x7f267c5bcdc0>\nusername   = 'miha'\n\nproject/tests/test_ui.py:347: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <ui.pages.reg_page.RegPage object at 0x7f267c652c10>, username = 'miha'\nemail = 'boris32@mail.ru', password = 'HJ2yCg)zh@'\nhome_page_object = <ui.pages.home_page.HomePage object at 0x7f267c652670>\nwrong_password = 'HJ2yCg)zh@qwe'\n\n    def sign_up(self, username, email, password, home_page_object, wrong_password=None):\n        self.paste_sign_up_info(username, email, password, wrong_password=wrong_password)\n        self.click_accept_reg_button()\n        self.click_submit_button()\n        try:\n            if wait(func=self.check_available_of_element(self.locators.POP_UP_ERR_FIELD, auto_use=False), timeout=1):\n>               raise InvalidRegInfoException(self.find(self.locators.POP_UP_ERR_FIELD).text)\nE               ui.utils.exceptions.InvalidRegInfoException: {'username': ['Incorrect username length'], 'password': ['Passwords must match']}\n\nemail      = 'boris32@mail.ru'\nhome_page_object = <ui.pages.home_page.HomePage object at 0x7f267c652670>\npassword   = 'HJ2yCg)zh@'\nself       = <ui.pages.reg_page.RegPage object at 0x7f267c652c10>\nusername   = 'miha'\nwrong_password = 'HJ2yCg)zh@qwe'\n\nproject/ui/pages/reg_page.py:31: InvalidRegInfoException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <test_ui.TestUI object at 0x7f267c5bcdc0>\nget_fake_user_info = ('mihalovdanila', 'HJ2yCg)zh@', 'boris32@mail.ru')\nlen_username = 4\nhome_page_object = <ui.pages.home_page.HomePage object at 0x7f267c652670>\nreg_page_object = <ui.pages.reg_page.RegPage object at 0x7f267c652c10>\n\n    @allure.feature('UI tests')\n    @allure.story('Sign up page story')\n    @pytest.mark.parametrize('len_username', [1, 2, 3, 4, 5])\n    def test_sign_up_negative_8(self, get_fake_user_info, len_username, home_page_object, reg_page_object):\n        \"\"\"Проверка регистрации пользователя с одновременно несовпадающими полями password, confirm_password и невалидным полем username\"\"\"\n        self.logger.start_logging(\n            f'./project/app_logs/{inspect.getframeinfo(inspect.currentframe()).function}_ui_log_{len_username}.txt')\n        with allure.step('Получаем информацию о невалидном пользователе'):\n            username, password, email = get_fake_user_info\n            if len(username) < len_username:\n                username *= len_username\n            username = username[:len_username]\n        with allure.step('Получаем объект страницы регистрации'):\n            reg_page = self.login_page.sign_up(reg_page_object)\n        with allure.step('Регистрируемся и проверяем валидное отбражение ошибки на странице регистрации'):\n            try:\n                reg_page.sign_up(username, email, password, home_page_object, wrong_password=password + 'qwe')\n            except InvalidRegInfoException as exception:\n                allure.attach(self.driver.get_screenshot_as_png(), 'Error screen', allure.attachment_type.PNG)\n                self.logger.stop_logging()\n                allure.attach(self.logger.get_log_info(), 'App log', allure.attachment_type.TEXT)\n>               assert exception.args[0] == 'Incorrect username length and passwords must match'\nE               assert \"{'username':...must match']}\" == 'Incorrect us...ds must match'\nE                 - Incorrect username length and passwords must match\nE                 + {'username': ['Incorrect username length'], 'password': ['Passwords must match']}\n\nemail      = 'boris32@mail.ru'\nget_fake_user_info = ('mihalovdanila', 'HJ2yCg)zh@', 'boris32@mail.ru')\nhome_page_object = <ui.pages.home_page.HomePage object at 0x7f267c652670>\nlen_username = 4\npassword   = 'HJ2yCg)zh@'\nreg_page   = <ui.pages.reg_page.RegPage object at 0x7f267c652c10>\nreg_page_object = <ui.pages.reg_page.RegPage object at 0x7f267c652c10>\nself       = <test_ui.TestUI object at 0x7f267c5bcdc0>\nusername   = 'miha'\n\nproject/tests/test_ui.py:352: AssertionError"}, "description": "Проверка регистрации пользователя с одновременно несовпадающими полями password, confirm_password и невалидным полем username", "steps": [{"name": "Получаем информацию о невалидном пользователе", "status": "passed", "start": 1590595632815, "stop": 1590595632815}, {"name": "Получаем объект страницы регистрации", "status": "passed", "start": 1590595632815, "stop": 1590595632934}, {"name": "Регистрируемся и проверяем валидное отбражение ошибки на странице регистрации", "status": "failed", "statusDetails": {"message": "AssertionError: assert \"{'username':...must match']}\" == 'Incorrect us...ds must match'\n  - Incorrect username length and passwords must match\n  + {'username': ['Incorrect username length'], 'password': ['Passwords must match']}\n", "trace": "  File \"/media/qunity/Workspace/Python_projects/qa-project/project/tests/test_ui.py\", line 352, in test_sign_up_negative_8\n    assert exception.args[0] == 'Incorrect username length and passwords must match'\n"}, "attachments": [{"name": "Error screen", "source": "d1e53f84-05ae-4290-a0b8-8700ae36bd47-attachment.png", "type": "image/png"}, {"name": "App log", "source": "4eb0e3d8-90a7-48e9-a014-7091b5be8eae-attachment.txt", "type": "text/plain"}], "start": 1590595632934, "stop": 1590595634749}], "parameters": [{"name": "len_username", "value": "4"}], "start": 1590595632751, "stop": 1590595634749, "uuid": "62f146e0-e14f-4e43-8fc1-17760fb6ca01", "historyId": "4814d30b9765f40ded6f27649255f721", "testCaseId": "b16ff2d1f506ca04887854c5a80bcb6f", "fullName": "tests.test_ui.TestUI#test_sign_up_negative_8", "labels": [{"name": "story", "value": "Sign up page story"}, {"name": "feature", "value": "UI tests"}, {"name": "tag", "value": "UI"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_ui"}, {"name": "subSuite", "value": "TestUI"}, {"name": "host", "value": "dmaksimov-laptop"}, {"name": "thread", "value": "22057-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_ui"}]}